builtin/providers/aws/import_aws_iam_policy_test.go

```diff
@@ -0,0 +1,85 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/iam"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+var testAccAwsIamPolicyConfig = `
+resource "aws_iam_policy" "test" {
+    name = "test_policy"
+    path = "/"
+    description = "My test policy"
+    policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "ec2:Describe*"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    }
+  ]
+}
+EOF
+}
+`
+
+func TestAccAWSIAMPolicy_importBasic(t *testing.T) {
+	resourceName := "aws_iam_policy.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSPolicyDestroy,
```

Comment:
This should jsut use the same Destroy method as defined in iam_user_test.go `testAccCheckAWSUserDestroy`